mbuild/utils/conversion.py

```diff
@@ -21,3 +21,268 @@ def RB_to_OPLS(c0, c1, c2, c3, c4, c5):
     f3 = -0.5 * c3
     f4 = -0.25 * c4
     return np.array([f1, f2, f3, f4])
+
+
+def RB_to_CHARMM(c0, c1, c2, c3, c4, c5):
+    """Converts Ryckaert-Bellemans (RB) type dihedrals to CHARMM type
+    or
+
+    RB_torsions = c0 + c1*Cos[Psi] + c2*Cos[Psi]^2 + c3*CosPsi]^3 + c4*Cos[Psi]^4 + c5*Cos[5*Psi]^5
+
+    where Psi= t-Pi = t - 180 degress
+
+    Parameters
+    ----------
+    c0, c1, c2, c3, c4, c5 : Ryckaert-Belleman coefficients (in kcal/mol)
+
+    converts to:
+
+    CHARMM_torsions =
+    = K0 * (1 + Cos[n0*(t) - (d0)] ) + K1 * (1 + Cos[n1*(t) - (d1)] ) + K2 * (1 + Cos[n2*(t) - (d2)] )
+    + K3 * (1 + Cos[n3*(t) - (d3)] )  +  K4 * (1 + Cos[n4*(t) - (d4)] )  + K5 * (1 + Cos[n5*(t) - (d5)] )  .
+
+    = K0 + K1 * (1 + Cos[n1*(t) - (d1)] ) + K2 * (1 + Cos[n2*(t) - (d2)] )
+    + K3 * (1 + Cos[n3*(t) - (d3)] )  +  K4 * (1 + Cos[n4*(t) - (d4)] )  + K5 * (1 + Cos[n5*(t) - (d5)] )  .
+
+    Returns
+    -------
+    0, K1, K2, K3, K4, K5, n0, n1, n2, n3, n4, n5, d0, d1, d2, d3, d4, and d5  : Charmm coefficients (in kcal/mol)
+
+    CHARMM_ dihedral coeffs : np.matrix, shape=(6,3)
+        Array containing the CHARMM dihedral coeffs  [[K0, n0, d0], [K1, n1, d1], [K2, n2, d2], [K3, n3, d3],
+        [K4, n4, d4], [K5, n5, d5]]  (in kcal/mol)
+
+    """
+    # see below or the long version is,  K0 = (c0 + c2 / 2 + 3 / 8 * c4) - K1 - K2 - K3 - K4 - K5
+    K0 = (c0  -c1 - c3 - c4/4 - c5)
+    K1 = (+c1 + 3/4 * c3 + 5/8 * c5)
+    K2 =  (+(1/2) * c2 + 1/2 * c4)
+    K3 =  (+(1/4) * c3 + 5/16 * c5)
+    K4 = (+(1/8) * c4)
+    K5 = (+(1/16) * c5)
+
+
+    n0 = 0
+    n1 = 1
+    n2 = 2
+    n3 = 3
+    n4 = 4
+    n5 = 5
+
+    d0 = 90
+    d1 = 180
+    d2 =  0
+    d3 =  180
+    d4 = 0
+    d5 = 180
+
+
+    return np.matrix([[K0, n0, d0], [K1, n1, d1], [K2, n2, d2], [K3, n3, d3], [K4, n4, d4], [K5, n5, d5]])
+
+
+
+
+
+#***********************************************
+# Converting base-10 to base-62 functions (start)
+#***********************************************
+
+def base10_to_base62_alph_num(base10_No):
+    '''Converst base 10 to base 62 so pdb/psf files can add may more than
+    9999 atoms and 999 residues.'''
+
+    '''base10_No = the base-10 number that you want to convert to base-62)'''
+
+    base62_No = 62
+    base10_No = int(base10_No)
+
+    whole_no =1
+    remainder = changeDigit_base10_to_base62_alph_num(int(base10_No % base62_No))
+    base62_Values =  str(remainder)
+    power =1
```

Comment:
These lines can be moved to a separate module like `mbuild/utils/encoding.py` or `mbuild/utils/basex.py`